client/my-sites/checkout/checkout-thank-you/google-apps-details.jsx

```diff
@@ -7,39 +7,94 @@ import i18n from 'i18n-calypso';
 /**
  * Internal dependencies
  */
-import { GOOGLE_APPS_LEARNING_CENTER } from 'lib/url/support';
+import { CONTACT, GOOGLE_APPS_LEARNING_CENTER } from 'lib/url/support';
 import PurchaseDetail from 'components/purchase-detail';
 import { useSelector } from 'react-redux';
 import { getCurrentUserEmail } from 'state/current-user/selectors';
+import { isGoogleApps } from 'lib/products-values';
+import { isGSuiteExtraLicenseProductSlug } from 'lib/gsuite';
 
-const GoogleAppsDetails = () => {
+const GoogleAppsDetails = ( { purchases } ) => {
 	const email = useSelector( getCurrentUserEmail );
 
+	const purchase = purchases.find( isGoogleApps );
+
+	if ( isGSuiteExtraLicenseProductSlug( purchase.productSlug ) ) {
+		return (
+			<PurchaseDetail
+				icon="mail"
+				title={ i18n.translate(
+					'Keep an eye on your email to finish setting up your new email addresses'
+				) }
+				description={ i18n.translate(
+					"We are setting up your new G Suite users but {{strong}}this process can take several minutes{{/strong}}. We will email you at %(email)s with login information once they are ready but if you still haven't received anything after a few hours, do not hesitate to {{link}}contact support{{/link}}.",
+					{
+						components: {
+							strong: <strong />,
+							link: (
+								<a
+									className="checkout-thank-you__gsuite-support-link"
+									href={ CONTACT }
+									rel="noopener noreferrer"
+									target="_blank"
+								/>
+							),
+						},
+						args: {
+							email,
+						},
+					}
+				) }
+				isRequired
+			/>
+		);
+	}
+
 	return (
 		<PurchaseDetail
 			icon="mail"
-			title={ i18n.translate( 'Check your email to finish setting up your G Suite account' ) }
-			description={ i18n.translate(
-				'We emailed you at {{strong}}%(email)s{{/strong}} with login information ' +
-					'so you can start using new professional email addresses and other G Suite apps. ' +
-					'If you can’t find it, try searching “G Suite” in your email inbox. {{link}}Learn more about G Suite{{/link}}',
-				{
-					components: {
-						strong: <strong />,
-						link: (
-							<a
-								className="checkout-thank-you__gsuite-support-link"
-								href={ GOOGLE_APPS_LEARNING_CENTER }
-								rel="noopener noreferrer"
-								target="_blank"
-							/>
-						),
-					},
-					args: {
-						email,
-					},
-				}
+			title={ i18n.translate(
+				'Keep an eye on your email to finish setting up your G Suite account'
 			) }
+			description={
+				<div>
+					<p>
+						{ i18n.translate(
+							'We are setting up your new G Suite account but {{strong}}this process can take several minutes{{/strong}}. We will email you at %(email)s with login information once it is ready, so you can start using your new professional email addresses and other G Suite apps.',
```

Comment:
I'm not a big fan of wrapping user-facing strings because it makes it more difficult to look them up later in OpenGrok. On the other hand, because of the presence of those `{{strong}}` tags near the beginning of the string searching for `We are setting up your new G Suite account but this process` would fail anyway, so I guess I could wrap them.